CHANGELOG

```diff
@@ -1,3 +1,51 @@
+pyqtgraph-0.12.2
+
+  Highlights
+    - Qt6 6.0 support has been removed in favor of Qt6 6.1+
+    - More numba utilization, specifically for makeARGB 
+    - Substantial ImageItem performance improvements have been made thanks to @pijyoi and @outofculture
+    - Significant performance improvements made to ScatterPlotItem and LinePlots
+    - More Expansive ColorMap features/support (more are coming!)
```

Comment:
remove expansive?   Otherwise, don't capitalize it.

Reply:
done!